samples/asgardeo-react-app/src/images/index.ts

```diff
@@ -0,0 +1,7 @@
+import Footer from './footer.png';
+import ReactLogo from './react-logo.png';
+
+export {
+    Footer,
+    ReactLogo
+}
```

Comment:
Let's not create a barrel file for this. instead let's directly import the image in relevant places

Reply:
Done